cypress.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'cypress'
```

Comment:
I think these alerts will popup for every PR which is annoying. even if they're silenced for this one. 
In a separate PR, we can add `cypress.config.ts` to the ignore list if we can't get fix these

Reply:
updated